pkg/artifactory/provider/framework.go

```diff
@@ -168,6 +168,7 @@ func (p *ArtifactoryProvider) Resources(ctx context.Context) []func() resource.R
 		user.NewArtifactoryManagedUserResource,
 		user.NewArtifactoryAnonymousUserResource,
 		security.NewArtifactoryGroupResource,
+		security.NewArtifactoryScopedTokenResource,
```

Comment:
Not blocker: I don't feel we nee the `Artifactory` part in the func name. I think we should remove that at some point.

Reply:
We use a similar naming in the SDK v2 provider, like `ResourceArtifactoryUser(),`, not sure about that.

Reply:
Well, I think it's also redundant there too 😄